packages/transition-frontend/src/components/forms/node/TransitNodeEdit.tsx

```diff
@@ -142,6 +143,47 @@ class TransitNodeEdit extends SaveableObjectForm<Node, NodeFormProps, NodeFormSt
             transitNodes1000mRadius: turfFeatureCollection(geojson.geometry ? [geojson] : []),
             transitNodesRoutingRadius: turfFeatureCollection(geojson.geometry ? [geojson] : [])
         });
+        serviceLocator.socketEventManager.emit(
```

Comment:
Is this the right place to put this code?

Reply:
NO, I moved it to componentDidMount and onValueChange